sidebarsDocs.js

```diff
@@ -72,6 +76,30 @@ module.exports = {
       ],
       collapsed: true,
     },
+    {
+      type: 'category',
+      label: 'Migration Guides',
```

Comment:
```suggestion
      label: 'Solana Migration Guides',
```
I think calling it "Solana Migration" will help discovery